spanner/value.go

```diff
@@ -49,13 +50,55 @@ const (
 	NumericScaleDigits = 9
 )
 
+// NumericLossHandlingMode describes the mode of how to deal with loss of
+// precision on numeric values.
+type NumericLossHandlingMode int
```

Comment:
I think it would be good to use the same name as in .NET: `LossOfPrecisionHandling`